setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-__version__ = "1.1.1-test"
+__version__ = "1.1"
```

Comment:
Shouldn't we follow semantic versioning like we do in other projects? In which case the version should be `1.1.0`.

Reply:
Unless you're just testing stuff and I should just wait till it's up for review (github doesn't say a PR is a draft when sending a notification so I missed that status :cry: )

Reply:
1.1 already exists; it was chosen by the previous maintainer. I don't know if PyPI, pip etc consider these equivalent.

This change is intended to revert to the status quo after my testing on test.pypi.org. I'll can leave a comment encouraging a bump to 1.1.1 for the next version?

Reply:
Makes sense, and a comment sgtm